public/app/frontend/src/components/navigation-main/navigation-main.html.twig

```diff
@@ -26,7 +26,7 @@ example:
         ]
     }%}
 #}
-<nav id="navigation-main" class="navigation-main">
+<nav id="navigation-main" class="navigation-main" aria-label="Primary navigation">
```

Comment:
This should be aria-label="Primary" otherwise the screen reader will say "Primary navigation, navigation"